tests/common-tests.h

```diff
@@ -3227,6 +3227,18 @@ TEST_CASE_METHOD(common_tests, "NULL with optional", "[core][boost][null]")
             CHECK(omi1.is_initialized() == false);
             CHECK(omi2.is_initialized());
             CHECK(omi2.get().get() == 125);
+
+
+            // test us bind with const vector<boost::optional>
```

Comment:
What does "us" mean here? Is it a typo?

Reply:
Us is mitake. Here must be "test use bind with const vector<boost::optional>"
I will fix